src/components/__tests__/Asana.test.tsx

```diff
@@ -0,0 +1,20 @@
+import renderer from 'react-test-renderer';
+
+import Asana, { IAsanaProps } from '../Asana';
+import moment from 'moment';
+import React from 'react';
+
+it('should find and render tasks when project is given', () => {
```

Comment:
These aren't really necessary for this project, but I appreciate the snapshot tests!

Reply:
Noted thanks!